irohad/consensus/yac/impl/yac_crypto_provider_impl.cpp

```diff
@@ -17,12 +17,20 @@
 
 #include "consensus/yac/impl/yac_crypto_provider_impl.hpp"
 #include "consensus/yac/transport/yac_pb_converters.hpp"
-#include "cryptography/ed25519_sha3_impl/internal/ed25519_impl.hpp"
-#include "cryptography/ed25519_sha3_impl/internal/sha3_hash.hpp"
+#include "cryptography/crypto_provider/crypto_signer.hpp"
+#include "cryptography/crypto_provider/crypto_verifier.hpp"
 
 namespace iroha {
   namespace consensus {
     namespace yac {
+      std::shared_ptr<shared_model::interface::Signature> createEmptySig() {
+        auto sig = shared_model::proto::SignatureBuilder()
+                       .publicKey(shared_model::crypto::PublicKey(""))
```

Comment:
Might be broken after stateless validation improvements

Reply:
Here we need an empty signature

Reply:
looks like workaround, is it really necessary?